src/helpers/configuration_prompt_helpers.sh

```diff
@@ -22,34 +22,38 @@ function ensure_knows_configuration {
   fi
 
   local main_branch_input
+  local main_branch_prompt='Please specify the main development branch by name or number'
 
   while [ -z "$main_branch_input" ]; do
     if [ "$(is_main_branch_configured)" = true ]; then
-      echo -n "Please specify the main development branch by name or number (current value: ${MAIN_BRANCH_NAME}): "
+      echo -n "$main_branch_prompt (current value: $(echo_inline_cyan_bold "$MAIN_BRANCH_NAME")): "
     else
-      echo -n "Please specify the main development branch by name or number (current value: None): "
+      echo -n "$main_branch_prompt (current value: None): "
     fi
```

Comment:
Thoughts on saving a main_branch_current_value (which would be the main branch in cyan or None) to reduce the duplication? Same for perennial branch prompts.